src/app/parser/parser.ts

```diff
@@ -0,0 +1,125 @@
+/* eslint-disable max-classes-per-file */
```

Comment:
Can we maybe disable this error in `.eslintrc.js`? It seems to be used in several files.